src/models/site/usePutSiteSettings.js

```diff
@@ -3,10 +3,10 @@ import queryKeys from '../../constants/queryKeys';
 
 export default function usePutSiteSettings() {
   return usePost({
-    url: '/site-settings/main/',
+    url: '/site-settings/data',
     deriveData: ({ data }) => data,
```

Comment:
This should be value: data,

Reply:
@Emily-Ke - this seemed to be working as-is, but let's give howard's suggestion a try and send data that way if it works. better to be consistent